### DISCUSSION TITLE
Moving from 8.1->11 Everything but 2 legacy dual HDHomeruns working?

### DISCUSSION THREAD

**Discussion by davidk21770** [OpenSageTV/Community#159] [Capture Devices]
Title says it all.
I am finally upgrading my computer and OS (only the computer's an upgrade ;-)).
From 8.1 -> 11.
I installed sage and migrated HDHomeRun.  Copied my old Program Files\SageTV to the new computer.  Checked the HD home runs using their setup apps and they streamed fine to VLC.
BUT, Sage will tune them (the correct lights come on), but it won't actually capture.
I checked and the drivers seem to be properly installed.
Perhaps it is because I stopped updating them with  20150604 ;-).
Since they are very vintage, what's a good version to hunt for and where (I don't use windows store and I doubt it'd give me the correct software for this old a unit and that's where the SD web site directs me).
Does anyone know of any issues / obstacles?

(And how do you search on this forum?  I seem to get everything no matter what I search for and I don't see a how-to?)

Thank You

-- I just realized that I've been using SageTV almost 20 years.  Wow!

**Comment by JustFred51** [comment]
HdHomeRun software/firmware version 20150604 isn't too old to run correctly; I'm still running (an even older) 20140121, which works just fine. 
The search function within forums.sagetv.com is no longer working, but a Google search of the old forums seems to work pretty well these days.  Example: search for "HdHomeRun site:sagetv.com" works.
 
Possible causes of the failure described:
1. Incorrect installation/config of the HDHomerun software.   Find detailed installation instructions here: https://sagetv.fandom.com/wiki/Appendix_Q:_Supported_Tuners_and_Capture_Devices and search within that page for "SiliconDust HDHomeRun Setup Instructions".  Pay particular attention to the instructions about the HdHomeRun Setup - Applications tab; this info applies to your situation with 20150604 software.
2. Win10/Win11 doesn't always initially work correctly with video capture software, attempting block the use of "cameras" in some situations (as a security issue).  I don't remember the details about how to tell Win10/11 to unblock the "camera".  Use Google to search sagetv.com

**Comment by davidk21770** [comment]
Resolved -- Thank you.  I needed to select an app and it wasn't finding 64 bit sagetv which I did look for.  I used JRiver Media Center :-).
